src/devtools/client/webconsole/components/FilterBar/ConsoleSettings.js

```diff
@@ -23,12 +23,10 @@ const { FILTERS } = require("devtools/client/webconsole/constants");
 class ConsoleSettings extends Component {
   static get propTypes() {
     return {
-      dispatch: PropTypes.func.isRequired,
```

Comment:
Are we not passing in dispatch anymore? It looks like it's still being used in the rest of the file and is still being passed from the parent (<FilterBar />)